src/main/target/CLRACINGF4AIR/target.c

```diff
@@ -23,15 +23,24 @@
  const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
 
 
-     { TIM4, IO_TAG(PB8), TIM_Channel_3, 1, IOCFG_AF_PP, GPIO_AF_TIM4,    TIM_USE_LED },                         // LED_STRIP
-     { TIM11, IO_TAG(PB9), TIM_Channel_1, 0, IOCFG_AF_PP, GPIO_AF_TIM11,    TIM_USE_PPM | TIM_USE_PWM },        // PPM
+     { TIM11, IO_TAG(PB9),      TIM_Channel_1, 0,  IOCFG_AF_PP, GPIO_AF_TIM11,    TIM_USE_PPM | TIM_USE_PWM },            // PPM
+	 { TIM4,  IO_TAG(PB8),      TIM_Channel_3, 1,  IOCFG_AF_PP, GPIO_AF_TIM4,     TIM_USE_LED },                          // LED_STRIP
```

Comment:
Indentation issue. Please use 4 spaces for indentation here (and in other places)